Mage/src/main/java/mage/abilities/hint/common/ConditionPermanentHint.java

```diff
@@ -0,0 +1,47 @@
+package mage.abilities.hint.common;
+
+import mage.abilities.Ability;
+import mage.abilities.condition.Condition;
+import mage.abilities.hint.ConditionHint;
+import mage.abilities.hint.Hint;
+import mage.game.Game;
+
+import java.awt.*;
+
+/**
+ * ConditionHint that is only shown on permanents.
+ *
+ * @author Susucr
+ */
+public class ConditionPermanentHint extends ConditionHint {
```

Comment:
Where did you end up using this?

Reply:
It was supposed to be the Hint for the cast ability of the Demon (It makes no sense to display an hint in another place than the battlefield).
Thanks for noticing that, I did forget to use that, will try if it actually works.

Reply:
In hand:
![image](https://github.com/magefree/mage/assets/34709007/e8a5f12c-3923-4a53-94c9-fc66f09f7462)
On battlefield, before cast:
![image](https://github.com/magefree/mage/assets/34709007/5d67b085-f64d-4189-9e44-974fb993d4f3)
On battlefield, after cast:
![image](https://github.com/magefree/mage/assets/34709007/1e71705a-7736-45dd-8efc-38c0cf85edc3)

Seems to work as intentended once used.

Reply:
Where's the "once during each of your turns" text for the ability itself though?